atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/progress/SweepProgressStoreImpl.java

```diff
@@ -149,15 +149,13 @@ private void tryInitialize() {
 
     private static Optional<SweepProgress> hydrateProgress(Map<Cell, Value> result) {
         if (result.isEmpty()) {
-            log.info("No persisted intermittent SweepProgress information found. "
-                    + "Sweep will choose a new table to sweep.");
+            log.info("No persisted SweepProgress information found.");
             return Optional.empty();
         }
         try {
             return Optional.of(OBJECT_MAPPER.readValue(result.get(CELL).getContents(), SweepProgress.class));
         } catch (Exception e) {
-            log.warn("Error deserializing SweepProgress object while attempting to load intermediate result. "
-                    + "Sweep will choose a new table to sweep.", e);
```

Comment:
Dont guess the caller ('while attempting to load intermediate result') and what might be done next.